python/pyspark/pandas/data_type_ops/base.py

```diff
@@ -206,3 +209,10 @@ def restore(self, col: pd.Series) -> pd.Series:
     def prepare(self, col: pd.Series) -> pd.Series:
         """Prepare column when from_pandas."""
         return col.replace({np.nan: None})
+
+    def isnull(self, series):
```

Comment:
Shall we add type annotations?

Reply:
The `series` parameter should be of `Union["Series", "Index"]`(`IndexOpsMixin`) type, shall we rename it more generally, for example, `index_ops`?
It would be wonderful the parameter type hints can be added as well.